### PR TITLE
[deployment] Celery worker

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

**Review by lbarcziova — APPROVED** [review]
LGTM :+1:

File: dist2src/worker/celer.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# SPDX-License-Identifier: MIT
+
+from os import getenv
+
+from celery import Celery
+from dist2src.worker.sentry import configure_sentry
+from lazy_object_proxy import Proxy
+
+
+class Celer:
```
Comment (lbarcziova) [inline dist2src/worker/celer.py:11]: This is named `Celer` on purpose?

Comment (jpopelka) [reply dist2src/worker/celer.py:11]: yes, there's already `celery` module with `Celery` class, so we need a different name for the file and class. Do you like `Celerizer` (as in p-s) more?

Comment (lbarcziova) [reply dist2src/worker/celer.py:11]: >  Do you like Celerizer (as in p-s) more?

it sounds better to me, but that's just a nitpick, I am ok with `Celer` as well

**Pull request merged**